Exygy/askdarcel-web
# Home Page Prep

# Pull Request Template

## Issue Overview

This PR addresses [#168](https://www.notion.so/exygy/Prep-Homepage-to-handoff-to-external-QA-00d56118eaa943918504f549ee848501)


## Description
This PR removes the previous sheltertech homepage and replaces it with our new components. Changes are made across all components to account for change in data fetching and styling.
- Hero
- Category Cards
- Opportunity Cards
- Event Cards
- Two Column Content

Note: Newsletter component is still needed 

## How Can This Be Tested/Reviewed?
Run local env and view home page. Should match up to [Figma designs here](https://www.figma.com/design/iDBeNK7SnZXF8wc3pbPTCY/Our415-Site?node-id=2372-3339&m=dev). There are a lot of diffs since I had to replace the previous home page, move all of the data fetching into the home page, and style each component to fit. Would probably be best to view home page first for visual issues then review code quality after

- [ ] Instead of fetching the data in each separate component, the data is now all fetched on the home page
- [ ] Wraps home page components with `HomePageSection` - content should have a max width and background color should expand full width
- [ ] Fixes issue with Two Column Content image disappearing on smaller breakpoints 
- [ ] Adds focus/active states to opp/event cards
- [ ] Review code quality
- [ ] Editing the home page fields in Sanity should immediately update the home page
https://our415.sanity.studio/development/structure/homePage;293c2d9c-0bba-4335-9532-3937f6dc9d86

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
